runtime/src/bank.rs

```diff
@@ -4053,6 +4062,13 @@ impl Bank {
             cost_tracker.set_limits(account_cost_limit, block_cost_limit, vote_cost_limit);
         }
 
+        if self
+            .feature_set
+            .is_active(&feature_set::raise_account_cu_limit::id())
+        {
+            self.apply_simd_0306_cost_tracker_changes();
```

Comment:
thanks for the reminder that cost tracker isn't persisted in snapshots :vomiting_face: